README.md

```diff
@@ -0,0 +1,97 @@
+[![Build Status][ci-img]][ci] [![Released Version][maven-img]][maven]
+
```

Comment:
Top level title is missing + description

Reply:
Done